docs/extensions/unreal_engine.md

```diff
@@ -0,0 +1,34 @@
+# Exporting your model to NVOL for Unreal Engine
+
+<p align="center">
+    <img width="800" alt="image" src="./imgs/desolation_unreal.png">
+</p>
```

Comment:
For images use this format
https://github.com/nerfstudio-project/nerfstudio/blob/1a2bbc5725261f617604de8d066cc1a8f33b6316/docs/quickstart/custom_dataset.md?plain=1#L158-L162